common-rendering/src/editorialPalette/background.ts

```diff
@@ -22,6 +22,17 @@ import { Colour } from '.';
 
 // ----- Functions ----- //
 
+const adSlot = (format: ArticleFormat): Colour => {
+	if (format.design === ArticleDesign.Comment ||
+	format.design === ArticleDesign.Letter ||
+	format.design === ArticleDesign.Editorial) {
+		return neutral[86];
+	}
+	return  neutral[97];
```

Comment:
Possible alternative, up to you whether you think it's more/less readable:

```suggestion
	switch (format.design) {
		case ArticleDesign.Comment:
		case ArticleDesign.Letter:
		case ArticleDesign.Editorial:
			return neutral[86];
		default:
			return  neutral[97];
	}
```